build.rs

```diff
@@ -23,28 +23,62 @@
 
 extern crate gcc;
 
+use std::env;
+use std::path::PathBuf;
+
+const ANDROID_AARCH64_COMPILER: &'static str = "toolchains/aarch64-linux-android-4.9/prebuilt/darwin-x86_64/bin";
+const ANDROID_ARM_COMPILER: &'static str = "toolchains/arm-linux-androideabi-4.9/prebuilt/darwin-x86_64/bin";
+const ANDROID_I686_COMPILER: &'static str = "toolchains/x86-4.9/prebuilt/darwin-x86_64/bin";
```

Comment:
I cannot guarantee that other ndk versions will compile the code. New ndk versions are released 2-3 times a year. We can update the script then